MH-FiorellaTheme5/source/partials/_info-product.erb

```diff
@@ -0,0 +1,88 @@
+<section class="product">
+  <div class="product-container-mobile">
+    <h2 class="product-title">Flower Bouquet Victoria</h2>
+    <img/>
```

Comment:
You have an image tag here without a source, why is that?

Reply:
I'm adding it on styles.css... should I change it?